src/focus-trap-react.js

```diff
@@ -45,6 +66,10 @@ class FocusTrap extends React.Component {
     if (this.props.paused) {
       this.focusTrap.pause();
     }
+
+    // if there is a _FocusTrapReact context from a parent focus trap, pause it
+    const {_FocusTrapReact: {pause} = {}} = this.context;
```

Comment:
Let's name this context prop something like `FocusTrapParent`, to clearly indicate that we're talking about a *parent*.